CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 ## Changelog
+### 0.14.4
+* [PR #334] Add support for building multiple architectures, specifically arm64.
+   * This now supports AWS Graviton based instances and enable Apple M1 based OS X systems as well.
```

Comment:
Actually it doesn't support M1. We'd need a CI system to build the `kinises_producer` for an M1. Since I didn't have that, I didn't add support, but it would be straight forward once that is available.

Reply:
Updated